content/en/docs/tasks/extend-kubernetes/setup-konnectivity.md

```diff
@@ -25,15 +25,8 @@ You need to configure the API Server to use the Konnectivity service
 and direct the network traffic to the cluster nodes:
 
 1. Make sure that
-the `ServiceAccountTokenVolumeProjection` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
-is enabled. You can enable
-[service account token volume protection](/docs/tasks/configure-pod-container/configure-service-account/#service-account-token-volume-projection)
-by providing the following flags to the kube-apiserver:
-   ```
-   --service-account-issuer=api
-   --service-account-signing-key-file=/etc/kubernetes/pki/sa.key
-   --api-audiences=system:konnectivity-server
-   ```
+[Service Account Token Volume Projection](/docs/tasks/configure-pod-container/configure-service-account/#service-account-token-volume-projection)
```

Comment:
This link (not added here) is broken.

Reply:
What do you mean it works to me

Reply:
If so then please ignore me. :) Link doesn't work from "Display the rich diff" view.

Reply:
@jkh52 that's a reference style link so it doesn't work when using the "Display the rich diff" view 😄